pkg/pool-manager/pool.go

```diff
@@ -146,7 +154,16 @@ func checkRange(startMac, endMac net.HardwareAddr) error {
 		}
 	}
 
-	return fmt.Errorf("Invalid range start: %s end: %s", startMac.String(), endMac.String())
+	return fmt.Errorf("Invalid range. rangeStart: %s rangeEnd: %s", startMac.String(), endMac.String())
+}
+
+func checkCast(mac net.HardwareAddr) error {
+	s := mac[0]
+	if s == 0x2 || s == 0x6 || s == 0xA || s == 0xE {
```

Comment:
Is it about octet though? AFAIK unicast is signalized by a single bit. Can't you just match that bit? As wiki says "When the least significant bit of an address's first octet is 0 (zero), the frame is meant to reach only one receiving NIC.". Could you & it with 1, if the result is 1, we are multicast. It will also make tests shorter.